src/epoch.rs

```diff
@@ -1540,6 +1596,65 @@ fn gpst() {
     assert!((epoch.as_gpst_days() + 5.0).abs() < EPSILON);
 }
 
+#[test]
+fn unix() {
+    use core::f64::EPSILON;
+    let now = Epoch::from_gregorian_utc_hms(2022, 5, 2, 10, 39, 15);
+    assert!(
+        now.as_tai_seconds() > now.as_utc_seconds(),
+        "TAI is not ahead of UTC."
+    );
+    assert!((now.as_tai_seconds() - now.as_utc_seconds() - 37.0).abs() < EPSILON);
+    assert!(
+        now.as_tai_seconds() > now.as_unix_seconds(),
+        "TAI is not ahead of UNIX."
+    );
+    assert!((now.as_unix_seconds() - 1651487955.0f64).abs() < EPSILON);
```

Comment:
Minor: I typically like the underscore to specify the type, it makes it easier to read.
```suggestion
    assert!((now.as_unix_seconds() - 1651487955.0_f64).abs() < EPSILON);
```

Reply:
Easy enough. I'll make sure I didn't add any more of these.